native/cocos/core/geometry/Frustum.h

```diff
@@ -140,12 +140,40 @@ class Frustum final : public ShapeBase {
         }
     }
 
+    Frustum(const Frustum &rhs) {
+        *this = rhs;
+    }
+
+    Frustum(Frustum &&rhs) {
+        *this = std::move(rhs);
+    }
+
     ~Frustum() override {
         for (auto *plane : planes) {
             plane->release();
         }
     }
 
+    // Can remove these operator override functions if not using Plane* in planes array.
+    Frustum &operator=(const Frustum &rhs) {
+        setType(rhs.getType());
+        for (size_t i = 0; i < planes.size(); ++i) { // NOLINT(modernize-loop-convert)
+            planes[i] = ccnew Plane();
+            planes[i]->addRef();
+            Plane::copy(planes[i], *(rhs.planes[i]));
+        }
+        return *this;
+    }
+
+    Frustum &operator=(Frustum &&rhs) {
+        setType(rhs.getType());
+        for (size_t i = 0; i < planes.size(); ++i) { // NOLINT(modernize-loop-convert)
+            planes[i] = rhs.planes[i];
```

Comment:
Need to release old planes
```c++
CC_SAFE_RELEASE(planes[i]);
planes[i] = rhs.planes[i];
```

Reply:
After looking at the source code of Frustum, I think planes should not be released.  Instead Plane needs a `reset` function to reset values to its original state.